wolfssl/wolfcrypt/sha512.h

```diff
@@ -112,6 +114,10 @@ enum {
 #else
 /* wc_Sha512 digest */
 typedef struct wc_Sha512 {
+#if defined(WOLFSSL_ESP32WROOM32_CRYPT) && \
```

Comment:
Please move this to the bottom of the structure. New members should be added at bottom. This ensures best backwards compatibility.

Reply:
Moved it to the bottom of the structure. Thank you for pointing out.